spec/features/request_a_visit_spec.rb

```diff
@@ -3,9 +3,7 @@
 RSpec.feature 'Booking a visit', js: true do
   include ActiveJobHelper
 
-  before do
-    create(:prison, name: 'Reading Gaol')
-  end
+  let!(:prison) { create(:prison, name: 'Reading Gaol') }
```

Comment:
If i recall correctly the default name for the prison returned by factory girl will be 'Reading Gaol'.